### PR TITLE
Render automatic deployment

### PR DESCRIPTION
**Pull request by nedkamburov**
– Configure secrets and credentials on the Render.com side
– Test deployment
– Change Readme to show the live demo link

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -74,5 +71,10 @@ group :test do
...
   gem "selenium-webdriver"
   gem "webdrivers"
-  gem "faker", "~> 2.23"
-end
\ No newline at end of file
+end
+
+gem "devise", "~> 4.8"
+gem "pundit", "~> 2.2"
+gem "faker", "~> 2.23"
```
Comment (igor04) [inline Gemfile:78]: it looks like `faker` need to be moved back to `developemnt`, `test`

File: Gemfile.lock [hunk 2]
```diff
@@ -245,6 +249,7 @@ GEM
 
 PLATFORMS
   x86_64-darwin-21
+  x86_64-linux
```
Comment (igor04) [inline Gemfile.lock:252]: better just have `ruby` - which means cross platform

File: app/views/admin/pages/home.html.erb [hunk 3]
```diff
@@ -16,12 +16,12 @@
...
       <div>
         <h4>All Users</h4>
-        <% for user in @users %>
-          <div>
-            <p><%= "#{ user.first_name } #{ user.last_name } (#{ user.role })" %></p>
-            <p><%= user.email %></p>
-            <hr>
-          </div>
-        <% end %>
+        <% @users.each { |user| %>
```
Comment (igor04) [inline app/views/admin/pages/home.html.erb:19]: please do not use `{}` for multiline blocks

Comment (nedkamburov) [reply app/views/admin/pages/home.html.erb:19]: Alright

**Pull request merged**